src/geocat/viz/taylor/__init__.py

```diff
@@ -1 +0,0 @@
-from .taylor import *
```

Comment:
@michaelavs When I look both into the code on my side and the branch on GitHub, those folders aren't there - could they just be in the PR since I deleted files from them? I'm not sure how else to get rid of them if that isn't the case.